src/read/mod.rs

```diff
@@ -384,7 +384,7 @@ pub enum Error {
 impl fmt::Display for Error {
     #[inline]
     fn fmt(&self, f: &mut fmt::Formatter) -> ::core::result::Result<(), fmt::Error> {
-        Debug::fmt(self, f)
+        write!(f, "{}", self.description())
```

Comment:
This is a user-visible change. I think this is fine for now, but I will file a follow-up issue.